qcodes/instrument_drivers/basel/sp983c.py

```diff
@@ -38,6 +39,13 @@ def __init__(self, name: str, **kwargs: Any):
                            vals=Enum(30., 100., 300., 1e3, 3e3, 10e3, 30e3,
                                      100e3, 1e6))
 
+        self.add_parameter('bias_volt',
```

Comment:
does this name correctly represent the "abstraction", as in, does the basel preamp know that this parameter is used for biasing, as opposed to something else? in other words, is the name of this parameter generic enough?

Reply:
instrument uses the word 'offset' instead of bias.. should it be changes to that?

Reply:
yes, i'd say it's better to reflect the term that the instrument uses because the cases on the user side may different and likely not for all of them the word "bias" would make sense.